src/Async/Qu.swift

```diff
@@ -0,0 +1,200 @@
+//
+//  Qu.swift
+//
+//  Created by Logan Wright on 10/24/15.
+//  Copyright © 2015 lowriDevs. All rights reserved.
+//
+
+public typealias Block = () -> ()
```

Comment:
If there's two libraries competing for the typealias, yes we'd have to replace them with a namespace if used.

The main reason I have it as public is that I'm using this to depict public argument types, so it kind of needs to be publicly exposed. Do you think I should just remove it and write all of the arguments out?